labs/lab9/9.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Lab 9 - Web Application
+layout: lab
+---
+
+## Overview
+
+We’re going to set up a flask app. And we are going to call it basicapp
+
+There is 3 main component to this lab. We will install Nginx, uWSGI, and flask through the apt command. Then we will git clone an example app and config files from class repository. Next, we will modify the configuration files. If configured correctly, we'll go through the startup procedure. Then end it with some debugging tips and tricks. 
+
+## Note
+
+* No sudo was included in any command examples. Use your best judgment to include sudo or not. 
```

Comment:
I'd remove this, it will just confuse users and break things when they `sudo` something and end up using the system python or something like that.

Reply:
Some parts do require sudo. Would they know to use it?